packages/preview/preavis/example.typ

```diff
@@ -0,0 +1,24 @@
+#import "lib.typ": lettre_preavis, locataire, adresse, proprietaire
+#lettre_preavis(
```

Comment:
The convention in Typst packages is to use kebab-case names for functions and their arguments. However, if you prefer, you can keep snake_case names.